.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,26 @@
+# .github/workflows/deploy-preview.yml
+
+name: Deploy to Preview Channel
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./example
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: subosito/flutter-action@v1
+      - run: flutter pub get
+      - run: flutter build web --release

```

Comment:
Minor suggestion:  Style-wise, I like the multi-line approach like:
```yaml
        run: |
          flutter pub get
          flutter build web --release
```

Reply:
Fixed!